src/utils/revision-utils.js

```diff
@@ -23,47 +23,43 @@ export function checkRevisionOnDocumentCreation({ newRevision, room, user }) {
   }
 
   if (!room) {
-    const userCanAssignAnyAccreditedEditor = hasUserPermission(user, permissions.MANAGE_ACCREDITED_EDITORS);
-    const userCanAssignSelfAsAccreditedEditor = hasUserPermission(user, permissions.PROTECT_OWN_DOC);
-    const userCanProtectOwnNewDoc = hasUserPermission(user, permissions.PROTECT_OWN_DOC);
-    const userCanProtectAnyDoc = hasUserPermission(user, permissions.PROTECT_ANY_DOC);
-    const userCanArchiveDoc = hasUserPermission(user, permissions.ARCHIVE_DOC);
-    const userCanVerifyDoc = hasUserPermission(user, permissions.VERIFY_DOC);
-    const userCanReviewDoc = hasUserPermission(user, permissions.REVIEW_DOC);
-
     const { accreditedEditors } = publicContext;
+    const userCanManagePublicContext = hasUserPermission(user, permissions.MANAGE_PUBLIC_CONTENT);
+    const userCanProtectOwnDocWhenCreating = hasUserPermission(user, permissions.PROTECT_OWN_PUBLIC_CONTENT);
+    const userCanDecideWhoCanProtectOwnDocWhenCreating = hasUserPermission(user, permissions.MANAGE_PROTECTORS_OF_OWN_PUBLIC_CONTENT);
```

Comment:
Technically that's not correct (like the permission name, see also my other comment): it should be rather `userCanFreelyModifyTheListOfAccreditedAuthorsForThisDocument`, but as stated in my other comment above, the simplifications in this PR would actually lend to just bundle this ability under `MANAGE_PUBLIC_CONTENT` (and therefore not needing this third boolean flag at all).

Reply:
Agreed 🤝